.github/workflows/style_check.yml

```
@@ -0,0 +1,38 @@
+name: Style check
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  # Trigger on pull requests.
+  pull_request:
+
+  # Trigger on pushes master
+  push:
+    branches:
+      - "master"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+jobs:
+  clang-tidy:
+    name: Check C++ Code Style
+    runs-on: ubuntu-latest
+    container:
+      image: glotzerlab/ci:2021.11-clang10_py38
```

We could bump this to [2022.05-clang10_py38](https://hub.docker.com/layers/ci/glotzerlab/ci/2022.05-clang10_py38/images/sha256-ec335538528cdd482f74d531ea1503225fa337267965849d771371c9b9453a9c?context=explore) while we're editing this.
```suggestion
      image: glotzerlab/ci:2022.05-clang10_py38
```